test/utils/misc.test.js

```diff
@@ -16,86 +23,86 @@ describe('misc', () => {
 
   // this is the yargs object created with cli command: mm-snap init
   const defaultArgv = {
-    _: ['init'],
-    verboseErrors: false,
-    v: false,
+    '_': ['init'],
+    'verboseErrors': false,
```

Comment:
can i vote again to remove this quote keys thing lol

Reply:
You can create an issue on https://github.com/MetaMask/eslint-config 😜

Reply:
caputed here: https://github.com/MetaMask/eslint-config/issues/175